fpsim/locations/ethiopia/mix_ethiopia.csv

```diff
@@ -0,0 +1,10 @@
+"","method","Freq","perc"
+"1","BTL/vasectomy",0.00679963735267452,0.679963735267452
+"2","condoms",0.0099728014505893,0.99728014505893
+"3","implant",0.308250226654578,30.8250226654578
+"4","injectables",0.504079782411605,50.4079782411605
+"5","IUD",0.028558476881233,2.8558476881233
```

Comment:
What a skew! I knew it was really skewed toward injectables and implants but seeing it in numbers is pretty wild.